accounts/abi/unpack_test.go

```diff
@@ -957,7 +956,7 @@ func TestOOMMaliciousInput(t *testing.T) {
 		}
 		encb, err := hex.DecodeString(test.enc)
 		if err != nil {
-			t.Fatalf("invalid hex: %s" + test.enc)
+			t.Fatalf("invalid hex: %s", test.enc)
```

Comment:
```suggestion
			t.Fatalf("decoding hex, error %v; input = %q, err, test.enc)
```

All errors originate from [this function](https://cs.opensource.google/go/go/+/refs/tags/go1.23.6:src/encoding/hex/hex.go;l=87) and explain what exactly was invalid. Using `%q` makes it easier to see empty strings.